DependencyInjection/Configuration.php

```diff
@@ -12,11 +12,10 @@ class Configuration implements ConfigurationInterface
      *
      * @return TreeBuilder
      */
-    public function getConfigTreeBuilder()
+    public function getConfigTreeBuilder(): TreeBuilder
     {
         $treeBuilder = new TreeBuilder('knp_markdown');
-        // BC layer for symfony/config < 4.2
-        $rootNode = method_exists($treeBuilder, 'getRootNode') ? $treeBuilder->getRootNode() : $treeBuilder->root('knp_markdown');
+        $rootNode = $treeBuilder->getRootNode();
```

Comment:
Hm, the comment says it was BC layer for <4.2... however, we still require Symfony `^4.0`  in composer.json so looks like that BC layer still relevant

Reply:
Symfony 4.0 was released 5 years ago, and has been officially EOL for 3 years (Jan, 2019). Anyone using it should lock to an old commit.  Actually, they should upgrade.